.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Package and Publish to Visual Studio Marketplace
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```

Comment:
Why not 18?

Reply:
or latest LTS?

Reply:
Not sure about the syntax for latest, but I'm sure I can find out. Which do you want me to use, 18 or latest?

Reply:
Unresolving as latest LTS is 20, not 16. If it doesn't work with 20 then could use 18 but 16 is no longer supported.

Reply:
Line 21 had/has already been changed to "node-version: 'lts/*'" as requested by Hossein.

Reply:
Oh! Sorry, I didn't see the 'Outdated' tag.